app/views/monthly_statistics/show.html.erb

```diff
@@ -85,112 +90,139 @@
+      (ITT2022) and include deferred applications from the <%= @current_cycle_name %> recruitment cycle (ITT2021).</p>
+    <p class="govuk-body">PNew data for each month in the ITT2022 recruitment cycle is added monthly.</p>
   </div>
 </div>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <h2 class="govuk-heading-l" id="by-age-group">3. By age group</h2>
-    <p class="govuk-body">The age of each applicant is calculated as of 31 August 2022.</p>
+    <h2 class="govuk-heading-l" id="data-by-category">Data by category</h2>
   </div>
 </div>
-<%= render CandidateInterface::MonthlyStatisticsTableComponent.new(caption: 'Candidates by age group', statistics: @statistics['by_age_group']) %>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <h2 class="govuk-heading-l" id="applications-by-sex">4. By sex</h2>
-    <p class="govuk-body">Something to explain how this is counting applications not applicants...</p>
+    <h3 class="govuk-heading-m" id="by-status">1. Application status</h3>
+    <p class='govuk-body'>When applying for the first time in a recruitment cycle, candidates can make up to 3 applications
+      at the same time.</p>
+    <p class='govuk-body'>The ‘first application’ column in the first table below counts only the status of their most
+       “favourable” application, if they had more than one application in progress.</p>
+    <p class='govuk-body'>So if a candidate is recruited to one course but rejected from another, only their recruited
+       application is counted.</p>
+    <p class='govuk-body'>“Favourable” is defined by the following order for the purpose of this data:</p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>recruited (most “favourable”)</li>
+      <li>offer accepted by the candidate, but they still need to meet the conditions of the offer</li>
+      <li>offer made by provider, but the candidate has not responded yet</li>
+      <li>awaiting provider decisions</li>
+      <li>offer declined by candidate</li>
+      <li>application withdrew by candidate</li>
+      <li>application rejected by provider (least “favourable”)</li>
+    </ul>
+
+    <p class='govuk-body'>Candidates can make one application at a time, as many times as they like, if their first set
+      of applications do not lead to a place.</p>
+    <p class='govuk-body'>The ‘apply again’ column of the first table below counts only the status of a candidate's most
+       recent ‘apply again’ application. Their previous applications are not counted anywhere in the table.</p>
+    <p class='govuk-body'>The status of any one application changes throughout its lifecycle. Statuses are captured at
+      the end of the data period.</p>
+    <%= render CandidateInterface::MonthlyStatisticsTableComponent.new(caption: 'Candidates by most “favourable” application status', statistics: @statistics['candidates_by_status']) %>
```

Comment:
`This table counts all applications by their status.` is missing between the tables